drivers/src/memory_layout.rs

```diff
@@ -55,26 +38,15 @@ pub const ROM_NSTACK_ORG: u32 = 0x5001FC00;
 //
 // Memory Sizes In Bytes
 //
+pub const PERSISTENT_DATA_SIZE: u32 = 53 * 1024;
+
 pub const ROM_RELAXATION_PADDING: u32 = 4 * 1024;
 pub const ROM_SIZE: u32 = 48 * 1024;
 pub const MBOX_SIZE: u32 = 128 * 1024;
 pub const ICCM_SIZE: u32 = 128 * 1024;
 pub const DCCM_SIZE: u32 = 128 * 1024;
 pub const ROM_DATA_SIZE: u32 = 996;
-pub const MAN1_SIZE: u32 = 6 * 1024;
-pub const MAN2_SIZE: u32 = 6 * 1024;
-pub const FHT_SIZE: u32 = 2 * 1024;
-pub const LDEVID_TBS_SIZE: u32 = 1024;
-pub const FMCALIAS_TBS_SIZE: u32 = 1024;
-pub const RTALIAS_TBS_SIZE: u32 = 1024;
-pub const PCR_LOG_SIZE: u32 = 1024;
-pub const MEASUREMENT_LOG_SIZE: u32 = 1024;
-pub const FUSE_LOG_SIZE: u32 = 1024;
-pub const DPE_SIZE: u32 = 5 * 1024;
-pub const PCR_RESET_COUNTER_SIZE: u32 = 1024;
-pub const AUTH_MAN_IMAGE_METADATA_MAX_SIZE: u32 = 7 * 1024;
-pub const IDEVID_CSR_SIZE: u32 = 1024;
-pub const DATA_SIZE: u32 = 27 * 1024;
+pub const DATA_SIZE: u32 = 8 * 1024;
```

Comment:
Do we need a DATA_SIZE separately defined?

Reply:
It's used by the linker script generator in https://github.com/chipsalliance/caliptra-sw/blob/main/cpu/gen/src/lib.rs#L24. 

IIRC there is some data that exists in the data section, e.g. the external vector interrupt table.